src/GitHub.App/ViewModels/Dialog/Clone/RepositoryCloneViewModel.cs

```diff
@@ -232,7 +239,8 @@ string ValidatePath(IRepositoryModel repository, string path)
         {
             if (repository != null)
             {
-                return service.DestinationExists(path) ?
+                // TODO: Update wording for DestinationAlreadyExists
```

Comment:
Does this TODO need addressing before we merge?

Reply:
Yes. I was holding off because the the internationalization work. I'll update it now.

Reply:
Changed to "A file exists at the destination path".